doc/versionHistory.md

```diff
@@ -1,5 +1,10 @@
 # Version History
 
+0.2.1
+
+- Create sendEvt.vi to write the event to TCP Client.
+- Modify sendCmdStatus.vi to hide user event instance to the user.
```

Comment:
Please consider to use **sendCmdStatus.vi** or `sendCmdStatus.vi` to express the vi.

Reply:
Done.